docker/build-project.sh

```diff
@@ -25,6 +25,12 @@ cd $WORKSPACE
 # get git commit ID of the project for using in buildhistory tagging
 CI_GIT_COMMIT=$(git rev-parse HEAD)
 
+# prepare git config
+#NOTE: bitbake does not pass git enviornment to tasks(do_patch),
+# hence we need to setup the git configuration 
+git config --global user.name "$GIT_COMMITTER_NAME"
+git config --global user.email "$GIT_COMMITTER_EMAIL"
+
 # FIXME: undbound variables used without checking:
```

Comment:
Paul's fix your are referring(http://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/?id=4b4387455c62cf19fa3d215a11b5d0b1211d4570) does not fix kernel patching issue, that address only PATCHTOOL='git' case.

This is the fix we needed:
http://lists.openembedded.org/pipermail/openembedded-core/2016-October/127230.html